src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -75,10 +80,39 @@ public CommandResult addPersonTags(Model model) throws CommandException {
         personToEdit.setCommonModules(userModuleTags);
 
         model.updateObservablePersonList();
+
+        //
+        // need to take the set of module tags in this command
+        // loop through and then put into the
+        for (ModuleTag tag : moduleTags) {
+            String day = tag.getDay();
+            String start = tag.getStartTime();
+            String end = tag.getEndTime();
+            if (day == null || start == null || end == null) {
+                continue;
+            }
+
+            Module mod = new Module(tag.toString());
+            int startHour = Integer.parseInt(start.substring(0, 2));
+            int startMin = Integer.parseInt(start.substring(2));
+            int endHour = Integer.parseInt(end.substring(0, 2));
+            int endMin = Integer.parseInt(end.substring(2));
+            LocalTime startTime = new LocalTime(startHour, startMin);
+            LocalTime endTime = new LocalTime(endHour, endMin);
```

Comment:
It is better to abstract out this function and make it `private` or `protected`. Adhere to Single Responsibility Principle for this method.